### ISSUE TITLE
[BUG] Navigation cancelled from "/"

### ISSUE THREAD

**Issue by Aksoom-Hussain** [TiagoSilvaPereira/vemto-issues#46]
```
Error: Navigation cancelled from "/" to "/projects/3/schema" with a new navigation.
    at We (app://./js/chunk-vendors.6bc08ad9.js:202:17387)
    at Ue (app://./js/chunk-vendors.6bc08ad9.js:202:17121)
    at f (app://./js/chunk-vendors.6bc08ad9.js:202:21301)
    at r (app://./js/chunk-vendors.6bc08ad9.js:202:16712)
    at je (app://./js/chunk-vendors.6bc08ad9.js:202:16749)
    at app://./js/chunk-vendors.6bc08ad9.js:202:21649
    at r (app://./js/chunk-vendors.6bc08ad9.js:202:16703)
    at app://./js/chunk-vendors.6bc08ad9.js:202:16731
    at app://./js/chunk-vendors.6bc08ad9.js:202:21560
    at app://./js/chunk-vendors.6bc08ad9.js:202:18041
```

---

vemto 0.6.1
Electron 8.5.2
win32 10.0.19041
Locale: en-US

**Comment by TiagoSilvaPereira** [comment]
Hi @Aksoom-Hussain thank you for reporting it, I'll fix this issue for the next version

**Comment by graham-ch** [comment]
Also getting the exact same error when opening a project.

Open Vemto -> Open Project -> Error

Only happens opening a project for the first time.  Does not happen on subsequent project opening.

vemto 0.6.2
Electron 8.5.2
darwin 17.7.0
Locale: en-GB

**Comment by TiagoSilvaPereira** [comment]
@graham-ch thank you for reporting, I'll fix it on next Vemto version

**Comment by TiagoSilvaPereira** [comment]
Fixed for next coming Vemto version (0.6.3)
